spyder/plugins/variableexplorer/widgets/texteditor.py

```diff
@@ -78,7 +79,12 @@ def __init__(self, text, title='', font=None, parent=None, readonly=False):
         self.layout.addLayout(btn_layout)
 
         # Make the dialog act as a window
-        self.setWindowFlags(Qt.Window)
+        if sys.platform == 'darwin':
+            # See: https://github.com/spyder-ide/spyder/issues/12825

```

Comment:
Please change this to

    # See spyder-ide/spyder#12825

Reply:
Done.